From 2ce1e2204ab336e3fcb5bb296689f6ca2a14a9d2 Mon Sep 17 00:00:00 2001
From: Christophe Priouzeau <christophe.priouzeau@st.com>
Date: Mon, 6 Dec 2021 10:59:45 +0100
Subject: [PATCH] ARM: STM32MP15: support of Preemptible Kernel

Signed-off-by: Christophe Priouzeau <christophe.priouzeau@st.com>
---
 arch/arm/mach-stm32/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/mach-stm32/Kconfig b/arch/arm/mach-stm32/Kconfig
index 981450315..9c40cc4c6 100644
--- a/arch/arm/mach-stm32/Kconfig
+++ b/arch/arm/mach-stm32/Kconfig
@@ -46,6 +46,7 @@ if ARCH_MULTI_V7
 config MACH_STM32MP157
 	bool "STMicroelectronics STM32MP157"
 	select ARM_ERRATA_814220
+	select ARCH_SUPPORTS_RT
 	default y
 
 config MACH_STM32MP13
-- 
2.17.1

